sources/manticore-worker/replica.php

```diff
@@ -115,6 +122,13 @@ function checkIsJoinNodeReady($joinHost, $qlPort, $clusterName)
     $manticoreJson->startManticore();
 
     $manticore = new ManticoreConnector('localhost', $qlPort, null, -1);
+
+    $settings = $manticore->showSettings();
+    if (!isset($settings['searchd.data_dir'])){
+        Logger::error("Data dir option wasn't passed in searchd conf. Replication disabled");
```

Comment:
"Data dir" -> data_dir

"searchd conf" - replace with balancer.config.content to make it more clear?